src/core/libraries/np_trophy/np_trophy.cpp

```diff
@@ -394,30 +387,30 @@ int PS4_SYSV_ABI sceNpTrophyGetGroupInfo(OrbisNpTrophyContext context, OrbisNpTr
 
                 if (!currentTrophyGroupID.empty()) {
                     if (std::stoi(currentTrophyGroupID) == groupId) {
```

Comment:
I see you call std::stoi(currentTrophyGroupID) here. Can't the original value be retrieved as a number with as_int() ?